examples/DGmethods/ex_002_IOstrings.jl

```diff
@@ -0,0 +1,255 @@
+  # Calculating initial condition norm 
+#=  eng0 = norm(Q)
+  @info @sprintf """Starting
+  norm(Q₀) = %.16e""" eng0
+=#
+  # Set up the information callback
+  starttime = Ref(now())
+  cbinfo = GenericCallbacks.EveryXWallTimeSeconds(2, mpicomm) do (s=false)
+    if s
+      starttime[] = now()
+    else
+    #=  energy = norm(Q) =#
+      @info @sprintf("""Update
+                     simtime = %.16e
+                     runtime = %s
+                     """, ODESolvers.gettime(lsrk),
+                     Dates.format(convert(Dates.DateTime,
+                                          Dates.now()-starttime[]),
+                                  Dates.dateformat"HH:MM:SS"))
+    end
+  end
+  
+  # Setup VTK output callbacks
+  step = [0]
+    cbvtk = GenericCallbacks.EveryXSimulationSteps(1) do (init=false)
+    mkpath(OUTPATH)
+    outprefix = @sprintf("%s/dycoms_%dD_mpirank%04d_step%04d", OUTPATH, dim,
+                           MPI.Comm_rank(mpicomm), step[1])
+    @debug "doing VTK output" outprefix
+    writevtk(outprefix, Q, dg, flattenednames(vars_state(model,FT)), 
+             dg.auxstate, flattenednames(vars_aux(model,FT)))
+        
+    step[1] += 1
+    nothing
+  end
+    
+  solve!(Q, lsrk; timeend=timeend, callbacks=(cbinfo, cbvtk))
+
+end
+
+using Test
+let
+  MPI.Initialized() || MPI.Init()
+  mpicomm = MPI.COMM_WORLD
+  ll = uppercase(get(ENV, "JULIA_LOG_LEVEL", "INFO"))
+  loglevel = ll == "DEBUG" ? Logging.Debug :
+    ll == "WARN"  ? Logging.Warn  :
+    ll == "ERROR" ? Logging.Error : Logging.Info
+  logger_stream = MPI.Comm_rank(mpicomm) == 0 ? stderr : devnull
+  global_logger(ConsoleLogger(logger_stream, loglevel))
+  @static if haspkg("CUDAnative")
+      device!(MPI.Comm_rank(mpicomm) % length(devices()))
+  end
+ #   @testset "$(@__FILE__)" for ArrayType in ArrayTypes
+  for ArrayType in ArrayTypes
+    # Problem type
+    FT = Float32
+    # DG polynomial order 
+    N = 4
+    # SGS Filter constants
+    C_smag = FT(0.15)
+    LHF    = FT(115)
+    SHF    = FT(15)
+    C_drag = FT(0.0011)
+    # User defined domain parameters
+    Δx, Δy, Δz = 10, 20, 30
+    xmin, xmax = 0, 1000
+    ymin, ymax = 0, 2000
+    zmin, zmax = 0, 3000
+
+    grid_resolution = [Δx, Δy, Δz]
+    domain_size     = [xmin, xmax, ymin, ymax, zmin, zmax]
+
+      brickrange = (grid1d(xmin, xmax, elemsize=FT(grid_resolution[1])*N),
+                    grid1d(ymin, ymax, elemsize=FT(grid_resolution[2])*N),
+                    grid1d(zmin, zmax, elemsize=FT(grid_resolution[end])*N))
+      
+    zsponge = FT(0.75 * zmax)
+    
+    topl = StackedBrickTopology(mpicomm, brickrange,
+                                periodicity = (true, true, false),
+                                boundary=((0,0),(0,0),(1,2)))
+
+    problem_name = "IOstring_test"
+    dt = 0.00000001
+    timeend = 1*dt
+
+    #Create unique output path directory:
+    OUTPATH = IOstrings_outpath_name(problem_name, grid_resolution)
```

Comment:
Note that different MPI ranks may generate different folders for their data.  Not sure if this was intended or not.

Reply:
Nope, it was not intended. Thanks for the insight @lcw Let me change that.

Reply:
Lucas, my fault, I used `bors r+` instead of `bors try` and the code was merged. 

Would you mind reverting the merge that came with it? The revert should not affect anyone because they  two functions that are independent of the rest of the code.

Reply:
Since we don't allow pushing to the master branch.  I think you need to
create a pull request that reverts the changes you want reverted.

Reply:
@smarras79: If you are worried about making this mistake in the future you can use the github [label][1] `do-not-merge-yet` on your pull request which will prevent [bors][2] from merging the pull request.

Alternatively you can create a [draft pull request][3]

[1]: https://help.github.com/en/articles/applying-labels-to-issues-and-pull-requests
[2]: https://github.com/climate-machine/CLIMA/blob/4ed7aa5c79399775023d12104910908145961bb9/bors.toml#L7
[3]: https://github.blog/2019-02-14-introducing-draft-pull-requests/